ReactWindows/ReactNative/Modules/Clipboard/ClipboardModule.cs

```diff
@@ -0,0 +1,88 @@
+﻿using ReactNative.Bridge;
+using System;
+using DT = Windows.ApplicationModel.DataTransfer;
+
+namespace ReactNative.Modules.Clipboard
+{
+    /**

```

Comment:
Could we keep the comments consistent with C#-style doc comments, even on internal code?  This is just something we'd have to change if we made this class public.